submissions/bmukha/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,90 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/bmukha/a-tiny-JS-world
+   Web app: https://bmukha.github.io/a-tiny-JS-world/
+   */
+
+class Character {
+  constructor(parameters) {
+    ({
+      specie: this.specie,
+      name: this.name,
+      gender: this.gender,
+      legs: this.legs = 2,
+      saying: this.saying,
+      friends: this.friends = [],
```

Comment:
At the moment of instantiation `this` refers to not what you think it does.
`{species}` will extract property `species` from an object passed as a parameter.
In destructuring you only need to list properties you want to extract, providing default values where needed.
There is a pattern for "renaming" when destructuring, but it is not the case here

Reply:
@OleksiyRudenko Updated again, please, review.